app/controllers/api/v1/users_controller.rb

```diff
@@ -31,6 +31,9 @@ def create
 
         if user.save
           session[:user_id] ||= user.id
+          session[:session_token] ||= user.session_token
```

Comment:
We're storing the token or the hash on the session itself?
I think from security perspective the token shouldn't be logged or stored, it will be transmitted securely to the client which will provide in upon requesting.